app/_src/policies/introduction.md

```diff
@@ -1,6 +1,651 @@
+```
+
+{% warning %}
+Policies are namespaced scope and currently the namespace must be the one the control-plane is running in `{{site.mesh_namespace}}` by default.
+{% endwarning %}
+
+{% endtab %}
+{% endtabs %}
+
+### Spec
+
+The `spec` field contains the actual configuration of the policy.
+
+Some policies apply to only a subset of the configuration of the proxy.
+
+- **Inbound policies** apply only to incoming traffic. The `spec.from[].targetRef` field defines the subset of clients that are going to be impacted by this policy.
+- **Outbound policies** apply only to outgoing traffic. The `spec.to[].targetRef` field defines the outbounds that are going to be impacted by this policy
+
+The actual configuration is defined in a `default` map.
+
+For example:
+
+{% policy_yaml base-example %}
+```yaml
+type: ExamplePolicy
+name: my-example
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: Mesh
+      default: # Configuration that applies to outgoing traffic
+        key: value
+  from:
+    - targetRef:
+        kind: Mesh
+      default: # Configuration that applies to incoming traffic
+        key: value
+```
+{% endpolicy_yaml %}
+
+{% tip %}
+While some policies can have both a `to` and a `from` section, it is strongly advised to create 2 different policies, one for `to` and one for `from`.
+{% endtip %}
+
+Some policies are not directional and will not have `to` and `from`. Some examples of such policies are [`MeshTrace`](/docs/{{ page.version }}/policies/meshtrace) or [`MeshProxyPatch`](/docs/{{ page.version }}/policies/meshproxypatch).
+For example
+
+{% policy_yaml non-directional %}
+```yaml
+type: NonDirectionalPolicy
+name: my-example
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  default:
+    key: value
+```
+{% endpolicy_yaml %}
+
+All specs have a **top level `targetRef`** which identifies which proxies this policy applies to.
+In particular, it defines which proxies have their Envoy configuration modified.
+
+{% tip %}
+One of the benefits of `targetRef` policies is that the spec is always the same between Kubernetes and Universal.
+
+This means that converting policies between Universal and Kubernetes only means rewriting the metadata.
+{% endtip %}
+
+## Writing a `targetRef`
+
+`targetRef` is a concept borrowed from [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) its usage is fully defined in [MADR 005](https://github.com/kumahq/kuma/blob/master/docs/madr/decisions/005-policy-matching.md).
+Its goal is to select subsets of proxies with maximum flexibility.
+
+It looks like:
+
+```yaml
+targetRef:
+  kind: Mesh | MeshSubset | MeshService | MeshGateway
+  name: "my-name" # For kinds MeshService, and MeshGateway a name has to be defined
+  tags:
+    key: value # For kinds MeshSubset and MeshGateway a list of matching tags can be used
+  proxyTypes: ["Sidecar", "Gateway"] # For kinds Mesh and MeshSubset a list of matching Dataplanes types can be used
```

Comment:
`labels` is missing

Reply:
also `namespace` and `sencionName` can be specified in targetRef don't we want to include it here?